example/next.config.js

```diff
@@ -7,6 +7,12 @@ const nextConfig = {
     alchemyKey: process.env.ALCHEMY_KEY,
     walletConnectProjectId: process.env.WALLET_CONNECT_PROJECT_ID,
   },
+  /**
+   * @todo debug why this is needed before merging
```

Comment:
If you've root-caused this, can you be more explicit in the comment as to what is actually going wrong?

Reply:
I am still waiting for the response from WalletConnect team to provide further explanation and it's primary blocker to merge this PR. Don't want to ship a workaround w/o fully understanding what is going on here.